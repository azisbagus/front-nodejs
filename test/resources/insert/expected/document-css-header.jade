!!!
html
    head
        title Layout
        link(rel='stylesheet', href='bootstrap-2.0.4.css')

    body
        include index
