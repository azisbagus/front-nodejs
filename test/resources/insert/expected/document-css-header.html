<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8"/>
    <title>Karthik Viswanathan</title>
    <link rel="stylesheet" href="bootstrap-2.0.4.css" />
</head>
<body>
    <script src="http://karthikv.net/js/respond.min.js?v=1316246035"></script>
</body>
</html>
